web/js/modules/map/util.js

```diff
@@ -247,47 +247,42 @@ function promiseTileLayer(layer, extent, viewState, pixelRatio) {
     }
     projection = viewState.projection;
     i = 0;
-    if (layer.type === 'VECTOR_TILE') {
-      // No need to look up tiles, vectors can resolve ASAP
-      resolve();
-    } else {
-      renderer = new OlRendererCanvasTileLayer(layer);
-      tileSource = layer.getSource();
-      // tileSource.expireCache(projection);
-      tileGrid = tileSource.getTileGridForProjection(projection);
-      currentZ = tileGrid.getZForResolution(
-        viewState.resolution,
-        renderer.zDirection,
+    renderer = new OlRendererCanvasTileLayer(layer);
+    tileSource = layer.getSource();
+    // tileSource.expireCache(projection);
```

Comment:
```suggestion
```
Can remove if not for related future changes.

Reply:
Yeah. I wasn't sure if someone else might have put it there for a reason but I will take it out since there are no comments suggesting it will be incorporated in the future. It seems like it could definitely be a useful function for animation... Maybe to run before pre-caching starts 🤔 . Added to brain.db (low disk space 😬 ) for future use.